cardinal/system/system_inventory_system.go

```diff
@@ -93,28 +93,92 @@ func Take(tokens []string, playerID uint32, roomID uint32, ts *TokeniserSystem,
 
 					// Update the player entity
 					if err := cardinal.SetComponent[component.Player](world, types.EntityID(playerID), &player); err != nil {
-						world.Logger().Debug().Msgf("Error updating the room entity: %v, after taking an object in the inventory system", err)
+						world.Logger().Error().Msgf("Error updating the room entity: %v, after taking an object in the inventory system", err)
 					}
 
 					// Update the room entity
 					if err := cardinal.SetComponent[component.Room](world, types.EntityID(roomID), &room); err != nil {
-						world.Logger().Debug().Msgf("Error updating the room entity: %v, after taking an object in the inventory system", err)
+						world.Logger().Error().Msgf("Error updating the room entity: %v, after taking an object in the inventory system", err)
 					}
 
 					description = fmt.Sprintf("You picked up a %s.", object.ObjectName)
 					tok_err = 0
-					hasBeenPickedUp = true
+					PickedUp = true
 					break
 				}
 			}
 		}
 
 	}
 
-	if hasBeenPickedUp == false {
+	if PickedUp == false {
 		description = "Can't pick something that doesn't exists, right?"
 		tok_err = 0
 	}
 
 	return description, tok_err
 }
+
+func Drop(tokens []string, playerID uint32, roomID uint32, ts *TokeniserSystem, world cardinal.WorldContext) (string, uint8) {
+	var tok_err uint8
+	var dropped bool
+	var tok string
+	var description string
+	tok = tokens[1]
+
+	// Get the room
+	room, err := GetRoom(types.EntityID(roomID), world)
+	if err != nil {
+		world.Logger().Error().Msgf("Error getting the room with ID: %v, for the Inventory System %v", roomID, err)
+	}
+
+	// Get the player
+	player, err := GetPlayer(types.EntityID(playerID), world)
+	if err != nil {
+		world.Logger().Error().Msgf("Error getting the player wiht ID: %v, for the Inventory System: %v", playerID, err)
+	}
+
+	// Get the Object Type
+	ObjectType := ts.GetObjectType(tok)
```

Comment:
"objectType" just because we should be consistent

Reply:
done